documentation/src/main/asciidoc/topics/upgrading.adoc

```diff
@@ -166,7 +166,6 @@ To use `SingleFileStore` as a file-based cache store with Infinispan 13 or later
 <persistence passivation="false">
   <single-file-store shared="false"
                      preload="true"
-                     fetch-state="true"
                      read-only="false"/>
 </persistence>
 ----
```

Comment:
Could we also add a deprecation notice for `fetch-state` to the upgrade guide?

Also I think it's worth explaining this line from the Jira description: "instead always use the first non shared store that has BULK_READ allowed". I've been looking through the docs and schema but I'm not quite sure what this means as an alternative to `fetch-state`. How do you allow BULK_READ for a non-shared store?

Reply:
> Could we also add a deprecation notice for fetch-state to the upgrade guide?

Yes, for sure. I forgot to add it.

> Also I think it's worth explaining this line from the Jira description: "instead always use the first non shared store that has BULK_READ allowed". I've been looking through the docs and schema but I'm not quite sure what this means as an alternative to fetch-state. How do you allow BULK_READ for a non-shared store?

Let me clarify this as BULK_READ is actually an internal thing. But basically any store that implements publishEntries method and isn't write only. All of our stores qualify for this.